Dispatcher/Views/Base.lproj/Main.storyboard

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You have some constraints issues in the HomePage in regards to the "Top Headlines In Israel"

<img width="436" alt="Screen Shot 2023-04-24 at 16 46 47" src="https://user-images.githubusercontent.com/100192030/234015634-64878a2f-e2c7-4d6c-968f-28dbe283a66e.png">